.gitignore

```diff
@@ -1,8 +1,43 @@
-# custom files
-custom/
-!custom/plugins/example
-!custom/example.zsh
-
-# temp files directories
-cache/
-log/
+.DS_Store
```

Comment:
Hi @aaron-goshine ! 

In order to follow best practices, it would be better without adding any reference about IDE or something like this at .gitignore file 😉